features/yield-calculator/YieldCalculator.tsx

```diff
@@ -111,14 +111,14 @@ const YieldCalculator = () => {
                   onChange={(e) => setTokenPrice(e.target.value)}
                   variant="outlined"
                   inputProps={{ min: "0", max: "10", step: "0.01" }}
-                  helperText={`Enter the market observable price`}
+                  helperText={`Enter the market price`}
```

Comment:
I wanted to change this to "Enter the price of yUSD of $" as it is even more explicit what to do